pyiron_experimental/pystemjob.py

```diff
@@ -1,5 +1,5 @@
 import matplotlib.pyplot as plt
-from pyiron_base import TemplateJob
+from pyiron_base.job.template import TemplateJob
```

Comment:
I guess the old implementation is more consistent. We should import all classes from `pyiron_base` rather than its submodules.

Reply:
I changed it back to importing `TemplateJob` from `pyiron_base`.